Documentation/HTKToMRTKPortingGuide.md

```diff
@@ -26,6 +28,8 @@ Platform support components (e.g., Windows Mixed Reality Device Manager) must be
 
 Some events no longer have unique events and now contain a MixedRealityInputAction. These actions are specified in the Input Actions profile and mapped to specific controllers and platforms in the Controller Mapping profile. Events like `OnInputDown` should now check the MixedRealityInputAction type.
 
+Learn more about the [new input system here](/Input/Overview.md).
```

Comment:
same as above - would be great if you can link to this from already existing text so we avoid the "here" links